src/Storages/MergeTree/MergeTreeData.h

```diff
@@ -610,7 +610,7 @@ class MergeTreeData : public IStorage, public WithMutableContext
         }
 
         /// It's safe to get name of any part
-        const String & getPartName() const { return part->name; }
+        const String & getPartNameV1() const { return part->name; }
```

Comment:
Did not understand why this is needed ...

Reply:
Oops, I just replaced all usages...